notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1614 @@
+{
```

Comment:
<p>Missing link to TruncatedNormal</p><p> </p><p>You can link to https://num.pyro.ai/en/stable/distributions.html#truncated-distributions for <code>a bunch of truncated distributions</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>